src/libYARP_serversql/include/yarp/serversql/yarpserversql.h

```diff
@@ -28,6 +29,23 @@
 #  define yarpserversql_DEPRECATED_API_MSG(X) YARP_DEPRECATED_API_MSG(X)
 #endif
 
+class YarpServerThread : public yarp::os::Thread
+{
+public:
+    virtual ~YarpServerThread(){}
+    void configure(int inArgc, char** inArgv)
+    {
+        argc = inArgc;
+        argv = inArgv;
```

Comment:
If this is intended to be a public API, we should clarify that the array of strings pointed by `inArgv` should be valid for all the life of the `YarpServerThread`. This is obvious if `argc` and `argv` are the arguments of the `main`, but in general it is not true, so we should it at least mention this in the documentation (if this is in a public API).